x/auth/types/account_test.go

```diff
@@ -214,6 +215,28 @@ func TestModuleAccountJSON(t *testing.T) {
 	require.Equal(t, acc.String(), a.String())
 }
 
+func TestModuleAccountJSONPB(t *testing.T) {
+	pubkey := secp256k1.GenPrivKey().PubKey()
+	addr := sdk.BytesToAccAddress(pubkey.Address())
+	baseAcc := types.NewBaseAccount(addr, nil, 10, 50)
+	acc := types.NewModuleAccount(baseAcc, "test", "burner")
```

Comment:
```suggestion
	acc := types.NewModuleAccount(baseAcc, "test", types.Burner)
```
How about using `types.Burner`?